test/write_square2D_test.cpp

```diff
@@ -82,58 +81,58 @@ TEST_CASE("writeAscii_test")
 
     return contents;
   };
-
-  std::string filename = "2x3_test.vtu";
+  std::string filename = "testfiles/square_2D/test.vtu";
+ // std::string filename = "ascii.vtu";
```

Comment:
IMO it is better to remove commented code in `master`
otherwise this can be quite confusing when trying to understand it later